### DISCUSSION TITLE
Combining several PIDs on the same gauge

### DISCUSSION THREAD

**Discussion by MasterPIC** [agronick/aa-torque#54] [Ideas]
Hi,
We can actually set the generic gauge to show also minimum or maximum values (computed on the same PID).
What about combining different PIDs on the same gauge?
Just for example, I'd like to read mean trip fuel consumption by the red arrow, instant fuel consumption by the dial, total trip fuel on the bottom (in place of the min/max field). This combination would make possible to reduce the number of panels and to access several values at time...

Thanks again
Armando

**Comment by agronick** [comment]
I am working on making it so you can switch between the guages and a line graph which will put all 3 values on the same graph.
